feat(changelog): cockpit-changed-cockpit-feature-request-is 2023-08-22 (#2058)
scaleway/docs-content: changelog/august2023/2023-08-22-cockpit-changed-cockpit-feature-request-is.mdx

---
title: Cockpit feature request is completed
status: changed
author:

>     fullname: 'Join the #[xxx] channel on Slack.'

Reviewer: ```suggestion
    fullname: 'Join the #observability channel on Slack.'
```

    url: 'https://slack.scaleway.com'
date: 2023-08-22
category: observability
product: cockpit
---

> We are happy to announce that we closed one of the most wanted feature : https://feature-request.scaleway.com/posts/38/log-centralization-and-management at Scaleway (open since Jan. 21)

Reviewer: ```suggestion
We are happy to announce that we closed one of the [most wanted features]https://feature-request.scaleway.com/posts/38/log-centralization-and-management  at Scaleway (open since Jan. 21).
```

> Almost every Scaleway's products are available in Cockpit for metrics and/or logs. New features and capabilities will be added to the product. You can see what's already there or planned : https://www.scaleway.com/en/docs/observability/cockpit/reference-content/cockpit-limitations/#product-integration-into-cockpit

Reviewer: ```suggestion
Almost every Scaleway products are available in Cockpit for metrics and/or logs. New features and capabilities will be added to the product. Refer to our [product documentation](https://www.scaleway.com/en/docs/observability/cockpit/reference-content/cockpit-limitations/#product-integration-into-cockpit) to see what is already available or planned.
```

> For all future needs, we encourage you to open a specific feature request of what you need to integrate in Cockpit.

Reviewer: ```suggestion
For all future feature requirements, we encourage you to open a specific feature request of what you need to integrate into Cockpit.
```
